cg/cli/add.py

```diff
@@ -206,10 +206,10 @@ def family(
         LOG.error(f"{customer_id}: customer not found")
         raise click.Abort
 
-    for panel_id in panels:
-        panel_obj: models.Panel = status_db.panel(panel_id)
+    for panel in panels:
+        panel_obj: models.Panel = status_db.get_panel_by_abbreviation(panel)
```

Comment:
```Python
from cg.store.models import Panel
```
try to get rid of models. in the files, import other models if needed

Reply:
I'll do this in a separate PR.